### ISSUE TITLE
channelMessage not send

### ISSUE THREAD

**Issue by klopfdreh** [HenrikJoreteg/SimpleWebRTC#203]
Hi,

I'm using the current latest.js and the signal server found here: https://github.com/andyet/signalmaster

I tried to reproduce the demo but the event channelMessage is not fired:
```javascript
	// If a message is received the volume is going to be refreshed
	webrtc.on('channelMessage', function (peer, label, data) {
	    if (data.type == 'volume') {
	        showVolume(document.getElementById('volume_' + peer.id), data.volume);
	    }
	});
```
I used the following setup:

npm install socket.io@0.9.16 node-uuid@1.2.0 getconfig@0.3.0

Thanks a lot in advance!

The implementation does not rely on the markup - does it?

**Comment by fippo** [comment]
no, looks like a regression in latest.js. the one on simplewebrtc.com still seems to work

**Comment by klopfdreh** [comment]
Mh, what I did so far is to set a breakpoint into the function mentioned above and it is not called. The HTML content is applied if a user entered the room but there is no event fired. 

I would be very pleased if you could give me some hints how to debug the issue.

**Comment by fippo** [comment]
the datachannel is never created so those events are not fired. Caused by a missing '!!' in 
https://github.com/HenrikJoreteg/webrtcsupport/blob/master/index-browser.js#L27
patched version should be up soon. Thanks!

**Comment by klopfdreh** [comment]
I have to thank you! :-)

**Comment by fippo** [comment]
fixed in 1.13.1 and current master

**Comment by klopfdreh** [comment]
That was insane fast fix for the issue and it is working, now! Thank you very very much! :+1:
